infra/src/test/kotlin/fi/fta/geoviite/infra/ui/testgroup2/LinkingTestUI.kt

```diff
@@ -146,7 +146,7 @@ class LinkingTestUI @Autowired constructor(
         trackLayoutPage.clickAtCoordinates(locationTrackStartPoint)
 
         linkingBox.link()
-        waitAndClearToastByContent("Raide linkitetty ja vanhentuneen geometrian linkitys purettu.")
+        waitAndClearToast("linking-succeeded-and-previous-unlinked")
```

Comment:
Ei suoraan tämän tiketin asiaa mutta nyt kun näitä katselee niin osa käännösavaimista on muotoa `<jotainjotain>-msg` ja osa vaan `<jotainjotain>`. Jossain kohtaa näitä voisi ehkä yhdistellä

Reply:
Joo -msg kuulostaa täysin tarpeettomalta lisäsanalta, vähän niinkuin luokkanimessä olisi lopussa Class tai Data. 

Muutenkin tuossa voitaisiin sopia joku yhteinen käytäntö miten ne nimet muodostetaan. Oma fiilis olisi että noiden pitäisi olla substantiivejä, eli mielummin "linking-success" kuin "linking-succeeded". Toisekseen niiden pitäisi pyrkiä minimaalisesti kuvaamaan se paikka/tilanne eikä koko haluttu teksti käännettynä snake-caseen: "linkin.save-success" eikä "linking-went-well-and-its-all-fine-now".